lib/appdmg.js

```diff
@@ -420,6 +420,21 @@ module.exports = exports = function (options) {
    **
    **/
 
+  pipeline.addStep('Signing image', function (next) {
+    var codeSignOptions = global.opts['code-sign']
+    if (codeSignOptions && codeSignOptions['signing-identity']) {
+      var codeSignIdentity = codeSignOptions['signing-identity']
+      var codeSignIdentifier = codeSignOptions['identifier']
+      util.codesign(codeSignIdentity, codeSignIdentifier, global.target, next)
+    } else {
+      return next.step()
```

Comment:
This should probably be `.skip()`? I guess that this path is broken now... this is why we need ci :smile:

Reply:
Ran tests locally and they pass

Reply:
Hm, I'm confused it does fail with `TypeError: next.step is not a function` when I run it manually though. I'll push a fix.

Reply:
Nice!

Reply:
I guess I must have run the test against master of the fork and not he branch the work was done in 🤦‍♀️ .

Should really pick up the Travis branch to avoid these issues in the future....